src/solve.jl

```diff
@@ -1,8 +1,9 @@
 function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algType,
               timeseries_init=uType[], ts_init=tType[], ks_init=[];
               d_discontinuities=tType[], dtmax=tType(7*minimum(prob.lags)), dt=zero(tType),
-              kwargs...) where {uType,tType,isinplace,
-                                algType<:AbstractMethodOfStepsAlgorithm,lType}
+              saveat=tType[], tstops=tType[], save_idxs=nothing,
```

Comment:
But then do we ever gain anything by having double the solution? It's not like that makes it more memory efficient if the ODE solution is already saving everything.